src/main/java/hybrid/it/internship/library/web/controller/BookController.java

```diff
@@ -0,0 +1,72 @@
+package hybrid.it.internship.library.web.controller;
+
+import hybrid.it.internship.library.service.impl.BookService;
+import hybrid.it.internship.library.service.impl.RentService;
+import hybrid.it.internship.library.web.dto.BookDTO;
+import hybrid.it.internship.library.web.mapper.BookMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/book")
+public class BookController {
+
+    @Autowired
+    private BookService bookService;
+
+    @Autowired
+    private RentService rentService;
+
+    @Autowired
+    private BookMapper bookMapper;
+
+    @GetMapping
+    public ResponseEntity<List<BookDTO>> getBooks() {
+
+        List<BookDTO> bookDTOS = bookMapper.toDTO(bookService.getAll());
```

Comment:
better to do conversion from entity to dto in Service layer (so not in controller)